pages/create.xml

```diff
@@ -30,6 +30,49 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <p>Please follow the instructions given on the application after the creation of a new file with that procedure. After downloading and before committing
          the new file, always open it in your editor to make sure that it is valid and, if necessary (which it nearly always is!), edit it.</p>
 </div>
+         <div type="level1">
+            <head>Example workflow</head>
+            <p>It is not rare that a single unit of work will involve several commits to several branches of different repositories. Here is an example</p>
+            <list rend="ordered">
+               <item>The contributor starts to edit a manuscript record,</item>
+               <item>she thinks an identifier for a new Textual Unit is needed,</item>
+            <item> she      opens an issue, labeled Textual Unit Identification, ping relevant people and 
+               make sure it is the case 
+               (this helps also contributors who do not have enough permissions to 
+               create Textual Units in the app). In the while, there is no need to add a <att>ref</att> to
+            <gi>title</gi> in <gi>msItem</gi>, for example, and a comment can be left that there is a pending issue about the identification of the Textual Unit.</item>
+              <item> 
+                 If the result of the discussion is positive and a new records is needed,
+                 the editor or some other collaborator creates it in the app (it needs to, is a textual unit!),</item>
+              <item> 
+                 Because the Textual Unit record has been created in the app, it needs to be synced.
+                 Make a PR to Works repository and make sure it is promptly reviewed, corrected and merged. 
+                 Planning this 
+                 in the issue where you discussed the creation is a good idea.
+                 Please note that it really needs to be merged quickly, to keep the app in sync.
+              </item> 
+               <item>Anyone can now use the newly created 
+                  Textual Unit ID in  manuscript records.
+              </item> 
+               <item>Once she is ready, a PR for the manuscript can be done to Manuscript.
+               </item>
+               <item>During review it turns out it was not necessary to create a Textual Unit, alas. A collaborator reviewing this PR did not see the discussion and could not intervene before.
```

Comment:
Sorry, I'm still not sure I got it. I made an issue to create AmazingNewPrayer. Everyone agrees it should be created. I make the branch AmazingPrayerBranch, create AmazingNewPrayer through the app, put it in the new folder, commit it to AmazingPrayerBranch and make a PR to merge into master. The great new colleague who just joined now sees this and remembers seeing the prayer elsewhere, and convinces everyone not to create the record. Can AmazingNewPrayer now be deleted from AmazingPrayerBranch, or does the branch have to be merged into master, and then a new branch be opened to make a PR to delete AmazingNewPrayer ?

Reply:
if you have created it in the app, it has to be merged anyway.

Reply:
that is way discussing it before will actually speed up the process... and eventually, agree on merging before letting more than 5 minutes go by, so that the new comer does not have the time to come into the discussion late.

Reply:
Got it. Maybe this merging, already doomed for deletion, could be specified here as a list item?

Reply:
I understand the need for a speedy process, but we are essentially four persons, 3 of them with part time positions, whose working hours often do not overlap...

Reply:
well, the point here is really to avoid this from happening at all...

Reply:
I understand, but since plan B is already spelled out, it could be also as explicit as possibly. But this might be an idiosyncratic understanding problem from my part.

Reply:
the 5 minutes where an exageration, just to make the point, I know it is not likely that it will happen in 5 minutes, but at least as much as it is not likely that a fifth person will come after a discussion and the agreed creation of a record... oder?

Reply:
Jaja :)

Reply:
I also agree that my full time position is a waste for the project, but it is not my fault

Reply:
I see the problem of time investment a bit differently. When I create two PR, one for MS and one for new works occuring there, and assigning them to the same reviewers, they immediately have much more context + I have probably typed the text in the work record, so the reviewers quickly see the problem if there is any. Creating an issue means putting the whole context into that issue, typing the eventual text, mentioning the MSS involved so that the others may go and check, etc, which then has to be again eventually copypasted. So the slow down shall be considerate as far as I see it. For me it seems faster to create the PR, and then, if a reviewer finds the record is a doublet, still go on and merge it and then go on and delete it. But maybe it is just me.

Reply:
> I also agree that my full time position is a waste for the project, but it is not my fault

I hope you know I did not mean anything close to that!

For me, it can be merged.

Reply:
I think, again, that the time consumption here is just moved, not augmented, and if there is any increase of time usage, it is well justified by the correctness of the procedure and compensated by saving another day like today entirely gone for fixing such issues.